README.md

```diff
@@ -106,25 +106,30 @@ As it turns out, people usually find this document before they sign a contract w
 
 UPDATE after 2 years or so: nobody seems to actually track it, but now that we have more data, we know it takes approximately 90 minutes to set everything up, so in worst case we can compensate based on that value.
 
-### RT projects
+### RT / RTO projects
 
 Below you can find the list of RT projects alongside with their descriptions.
```

Comment:
Maybe sort the table by the project name?

Reply:
Also, to improve readability I would suggest to start each description with a verb and use consistent sentence structure, ie:

- Write articles or blog posts under the company name and present the company to the outside world.
- Develop our internal Django template.
- Discuss upcoming projects with existing clients or provide theoretical after-sales support on fixed-price projects.
- Organize and procure equipment and supplies needed in the company, including laptops and office supplies, or perform other virtual assistant tasks.
- Go through training materials during the onboarding phase.

I can commit all relevant changes if you agree.